python/cudf/cudf/tests/test_multiindex.py

```diff
@@ -1089,3 +1089,37 @@ def test_multiindex_sort_values(pmidx, ascending, return_indexer):
         actual = actual[0]
 
     assert_eq(expected, actual)
+
+
+@pytest.mark.parametrize(
+    "pdi",
+    [
+        pd.MultiIndex(
+            levels=[[1, 3.0, 4, 5], [1, 2.3, 5]],
+            codes=[[0, 0, 1, 2, 3], [0, 2, 1, 1, 0]],
+            names=["x", "y"],
+        ),
+        pd.MultiIndex(
+            levels=[[1, 3, 4, -10], [1, 11, 5]],
+            codes=[[0, 0, 1, 2, 3], [0, 2, 1, 1, 0]],
+            names=["x", "y"],
+        ),
+        pd.MultiIndex(
+            levels=[["a", "b", "c", "100"], ["1", "100", "5"]],
+            codes=[[0, 0, 1, 2, 3], [0, 2, 1, 1, 0]],
+            names=["x", "y"],
+        ),
+    ],
+)
+@pytest.mark.parametrize("ascending", [True, False])
+def test_multiIndex_argsort(pdi, ascending):
+    gdi = cudf.from_pandas(pdi)
+
+    if not ascending:
+        expected = pdi.argsort()[::-1]
```

Comment:
Do we do this because Pandas `argsort()` doesn't have an `ascending` argument?

Reply:
Yes

Reply:
👍